lib/pact_broker/client/cli/matrix_commands.rb

```diff
@@ -39,6 +43,31 @@ def can_i_deploy(*ignored_but_necessary)
               exit(can_i_deploy_exit_status) unless result.success
             end
 
+            desc "can-i-merge", "Checks if the specified pacticipant version is safe to merge into the main branch."
```

Comment:
will docs be generated for this or will we need to add into the README when reviewed

Reply:
I probably just forgot to re-run the generate docs script after I updated that. Will do that.